druid/src/contexts.rs

```diff
@@ -574,7 +574,16 @@ impl_context_method!(EventCtx<'_, '_>, UpdateCtx<'_, '_>, LifeCycleCtx<'_, '_>,
         self.widget_state.needs_layout = true;
     }
 
-    /// Request an animation frame.
+    /// Request an [`AnimFrame`] event.
+    ///
+    /// Note that if you call this when handling an [`AnimFrame`] event,
```

Comment:
I think this applies in general, not just when handling an `AnimFrame` event: `request_anim_frame` doesn't invalidate anything and so if you're animating then you still need to invalidate. Maybe it's just me, but I find the "request a frame" / "invalidate a rect" distinction more natural than trying to distinguish between "animation frame" and "actual rendered frame"

Reply:
Yeah it's definitely general, my writing was just coming from a space where `MouseDown` never meant paint, but `AnimFrame` used to mean that (as the comment also said).

I'll see if I can adjust the wording a bit to be more precise about lack of invalidation being the issue.

Reply:
As another note, I actually ran into an issue with an older app of mine that used `AnimFrame` to measure render fps but the widget itself only painted when the fps changed. Thus the comment's purpose was to make sure it's documented that such tricks no longer work.